generators/client/templates/react/src/main/webapp/app/modules/account/password/password.tsx.ejs

```diff
@@ -37,7 +37,7 @@ export interface IUserSettingsState {
   password: string;
 }
 
-export class SettingsPage extends React.Component<IUserSettingsProps, IUserSettingsState> {
```

Comment:
can you also update the `IUserSettingsProps` and `IUserSettingsState` to something like `IUserPasswordProps/State`